### ISSUE TITLE
Day 4 CSS

### ISSUE THREAD

**Issue by fullstack-deploy** [fullstackreact/30-days-of-react#45]
Hi Nate,

Just some honest feedback. First I got stuck on 4th lesson with the CSS. Now I can probably solve it by focusing and spending some time on that. 
But I am looking for learning react and not doing debugging on css.

The other thing is that when you paste in code into your existing index.html there is no full overview how the end result should look at. The JSfiddle seems not to be connected to 4th lesson, because there is more advanced code example in it. So you miss out that you need to include <Timeline /> if you really want to see it on your browser. 

I am giving up now as I get a bad feeling the lessons will continue like this and makes me focus on other thing than learning the basics of react.

Thanks anyway for trying give people free lessons on React. But I think you should look over the lesson 4 again as it seems not to be working (old links to css, jsfiddle?)

-- Friend

**Comment by jashmenn** [comment]
There's no reason this file should be out in a gist -- it should just be included in the repository. The URL [in the post](https://github.com/fullstackreact/30-days-of-react/blob/master/day-04/post.md) should link to this Github repo

**Comment by Damiloju** [comment]
The file work, but something seems to be wrong with the css itself

**Comment by Damiloju** [comment]
> The file works, but something seems to be wrong with the css itself

The issue is with the demo class, I figured the elements were not wrapped in a demo class in the book

**Comment by diamondman** [comment]
I found that the class="demo" was missing thanks to a helpful stackoverflow poster, but even after that, it would not load correctly. I used a [CSS validator on it, and got the same error my browser was throwing](https://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fgist.githubusercontent.com%2Fauser%2F2bc34b9abf07f34f602dccd6ca855df1%2Fraw%2F070d6cd5b4d4ec1a3e6892d43e877039a91a9108%2Ftimeline.css&profile=css3svg&usermedium=all&warning=1&vextwarning=&lang=en) (my browser showed it as a warning, but did not load the CSS).

I then downloaded the file, saved it locally, and updated the reference. Then I saw styles loaded, but every style was applied to the div with the demo class. I am a CSS idiot, so I likely messed something up, but I had to grab the css file from [here](https://github.com/fullstackreact/30-days-of-react/blob/master/day-04/src/components/Timeline/Timeline.css) to make it work at all for me. For now, I recommend anyone with this issue just use this last link instead.
